tests/acceptance/receive_package_test.js

```diff
@@ -0,0 +1,39 @@
+import Ember from 'ember';
+import startApp from '../helpers/start-app';
+import { module, test } from 'qunit';
+import '../factories/offer';
+import FactoryGuy from 'ember-data-factory-guy';
+import TestHelper from 'ember-data-factory-guy/factory-guy-test-helper';
+
+var App, offer1, item1, package1;
+
+module('Receive package', {
+  beforeEach: function() {
+    App = startApp({}, 2);
+    TestHelper.setup();
+    item1 = FactoryGuy.make("item", {state: "accepted"});
+    offer1 = FactoryGuy.make("offer", { state: "received", items: [item1] });
+    package1 = FactoryGuy.make("package", { offerId: parseInt(offer1.id), state: "received", item: item1});
+  },
+  afterEach: function() {
+    Em.run(function() { TestHelper.teardown(); });
```

Comment:
I think here we need Ember instead of Em.

Reply:
Oops.. yes I will change it, thanks.

Reply:
@namrataukirde I checked it , It should be Em.run only. :)